src/index.ts

```diff
@@ -1,22 +1,19 @@
-import { createClient } from "@supabase/supabase-js";
 import { createActionsPlugin } from "@ubiquity-os/plugin-sdk";
 import { LogLevel } from "@ubiquity-os/ubiquity-os-logger";
-import { createAdapters } from "./adapters";
 import { run } from "./run";
-import { Context, SupportedEvents } from "./types/context";
+import { SupportedEvents } from "./types/context";
 import { Env, envSchema } from "./types/env";
 import { AssistivePricingSettings, pluginSettingsSchema } from "./types/plugin-input";
-import { Command } from "./types/command";
 
-createActionsPlugin<AssistivePricingSettings, Env, Command, SupportedEvents>(
+createActionsPlugin<AssistivePricingSettings, Env, null, SupportedEvents>(
```

Comment:
It makes sense to me to make null the last typing here so that it's optional to pass in, right?

Reply:
It is part of the SDK, so should require a change there. Rfc @whilefoo